Jenkinsfile

```
@@ -90,7 +90,7 @@ pipeline {
     stage('Push Image') {
       when{
         expression {
-          return env.GIT_BRANCH == 'origin/master';
+          return env.GIT_BRANCH == 'origin/v3.5.0';
```

```suggestion
          return env.GIT_BRANCH == 'origin/master';
```
This needs to reflect the correct branch which would be master in this case.